CMakeLists.txt

```diff
@@ -53,15 +53,15 @@ if(NOT CMAKE_BUILD_TYPE)
   set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS Debug Release RelWithDebInfo MinSizeRel)
 endif()
 
-option(PRECICE_MPICommunication "Enables MPI-based communication and running coupling tests." ON)
-option(PRECICE_PETScMapping "Enable use of the PETSc linear algebra library." ON)
-option(PRECICE_PythonActions "Python support" ON)
-option(PRECICE_Packages "Configure package generation." ON)
+option(PRECICE_MPI_COMMUNICATION "Enables MPI-based communication and running coupling tests." ON)
+option(PRECICE_PETSC_MAPPING "Enable use of the PETSc linear algebra library for global-iterative RBF data mappings." ON)
+option(PRECICE_PYTHON_ACTIONS "Enable Python support for preCICE actions." ON)
+option(PRECICE_CONFIGURE_PACKAGE_GENERATION "Configure package generation." ON)
 option(BUILD_SHARED_LIBS "Build shared instead of static libraries" ON)
 option(BUILD_TESTING "Build tests" ON)
 option(PRECICE_ALWAYS_VALIDATE_LIBS "Validate libraries even after the validatation succeeded." OFF)
-option(PRECICE_ENABLE_C "Enable the native C bindings" ON)
-option(PRECICE_ENABLE_FORTRAN "Enable the native Fortran bindings" ON)
+option(PRECICE_C_BINDINGS "Enable the native C bindings" ON)
+option(PRECICE_FORTRAN_BINDINGS "Enable the native Fortran bindings" ON)
```

Comment:
To more easily find these in ccmake, would it make more sense to name them as:

```
PRECICE_BINDINGS_C
PRECICE_BINDING_FORTRAN
```
?